### PR TITLE
add klusterletaddonconfig

### PR DESCRIPTION
**Pull request by SherinV**
Signed-off-by: Sherin Varughese <shvarugh@redhat.com>

**Related Issue:**  https://github.com/stolostron/backlog/issues/22013

### Description of changes
Add klusterletaddonconfig to view disabled cluster messages as part of the point below.
[ ] Revisit all hard coded queries in UI to make sure they work. (Ex: messages in UI)

### CODE REVIEW

**Comment by jlpadilla** [comment]
@SherinV Changes look good but missing unit tests.

**Comment by SherinV** [comment]
@jlpadilla Can we merge this?

File: pkg/transforms/klusterletaddonconfig_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/*
+IBM Confidential
+OCO Source Materials
+(C) Copyright IBM Corporation 2019 All Rights Reserved
+The source code for this program is not published or otherwise divested of its trade secrets,
+irrespective of what has been deposited with the U.S. Copyright Office.
+Copyright (c) 2020 Red Hat, Inc.
+*/
```
Comment (jlpadilla) [inline pkg/transforms/klusterletaddonconfig_test.go:8]: Remove this.

**Review by jlpadilla — APPROVED** [review]
/lgtm

**Pull request merged**